pom.xml

```diff
@@ -42,8 +42,10 @@
 
     <properties>
         <jenkins.version>1.580.1</jenkins.version>
-        <java.level>6</java.level>
+        <java.level>8</java.level>
```

Comment:
From what I can see you are only using the language level in the tests, so it would be better to rewrite them.
At some point someone should bump the core dependency so that it gets tested with something that people actually potentially use. But there doesn't seem to actually be needed in this PR.

Reply:
Ah, sorry - I'll look into rewriting those tests then